nlp/2021_06_18_data_augmentation/README.md

```diff
@@ -0,0 +1,10 @@
+# Data augmentation is all you need?
```

Comment:
I think the title is a bit too strong, since it's more like 'data is all you need' 😛 .

Perhaps something like 'transformer-based data augmentation tricks' or something?